documentation/modules/rn-2.6.adoc

```diff
@@ -58,9 +58,25 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of a static IP in a migration from vSphere
+
+You can now preserve a static IP when importing VMs that have vSphere as their source provider. link:https://issuesredhat.com/browse/MTV-1148[(MTV-1148)]
+
+.Entering a list of LUKS decryption passphrases in a vSphere migration plan
```

Comment:
this title and the description below make this look like it was possible to migrate LUKS-encrypted disks before by specifying a single decryption key and now it's possible specify multiple keys, while it wasn't possible to migrate VMs with LUKS-encrypted disks at all in previous versions - so this item needs to be rephrased

Edit: I think I see the source of this confusion - as the related Jira issues talk about changes in virt-v2v, however, we need to document the changes in MTV in this context

Reply:
Fixed.